terms.html

```diff
@@ -114,9 +123,12 @@
 The ability of a <a>holder</a> to make fine-grained decisions about what
 information to share.
   </dd>
-  <dt><dfn data-lt="subjects|subject's">subject</dfn></dt>
-  <dd>
-A thing about which <a>claims</a> are made.
+  <dt><dfn data-lt="subjects|subject's|subject(s)">subject</dfn></dt>
+The <a>entity</a> about which a <a>claim</a> is made.
```

Comment:
Subjects don't need to be entities.

Reply:
If the word entity does not apply to `subject`, `controller`, `issuer`, `verifier`... I am not sure why it is valuable to use `entity` instead of any of these existing roles.

Reply:
OR13, it does apply to controllers, issuers, and verifiers, who all take actions within the VC framework. 

Subjects, however, have no innate actions and don't need the ability to take actions just *as subjects*. As a result, Subjects only need to be referable in statements. They do not need to be "entities".

The challenge as writers is how do we say "Subjects are the subject of statements about them" without repeating the defined term.

Some options:
"Subjects are the focal point of claims made regarding them."
"Subjects are the focus of claims about them."
"Subjects are the focal point of claims concerning them."

Reply:
"Subjects are graph node identifiers that are the origin and target of graph edges".

Reply:
This should have been fixed now by using the RDF 'resource' concept.